INN/umbrella-citylimits
# Add new series header to old series template header

## Changes

This pull request makes the following changes:

- Adds our new header from the new series landing page template to the old series page template header in `header-rezone.php`
- Allows the styling for the series header/banner to be used even if the parent container is not `.cftl-tax-landing-template-series-landing-special-projects-php`

Main page:
![screencapture-citylimits-test-series-futuremap-2020-02-21-10_49_01](https://user-images.githubusercontent.com/18353636/75049277-ed860680-5497-11ea-944b-a6a9d5ad97da.png)

Interior page:
![screencapture-citylimits-test-neighborhoods-bay-street-2020-02-21-10_49_49](https://user-images.githubusercontent.com/18353636/75049265-e8c15280-5497-11ea-8d23-444d635fdffa.png)

Mobile:
![screencapture-citylimits-test-series-futuremap-2020-02-21-10_49_24](https://user-images.githubusercontent.com/18353636/75049276-eced7000-5497-11ea-9f99-9808ada6bb16.png)

## Why



For #147

## Testing/Questions

Features that this PR affects:

- Old series template header partial


Questions that need to be answered before merging:

- [ ] Could any of this be done better?
- [ ] Any design flaws that need to be fixed?

Steps to test this PR:

1. View a series landing page using the old template
2. View on frontend and make sure it looks ok
